bgpd/bgp_vty.c

```diff
@@ -14466,7 +14467,7 @@ DEFUN (show_community_list_arg,
 	if (argv_find(argv, argc, "ip", &idx)) {
 		vty_out(vty, "This config option is deprecated, and is scheduled for removal.\n");
 		vty_out(vty, "if you are using this please migrate to the below command.\n");
-		vty_out(vty, "'show bgp community-list <(1-500)|WORD>'\n");
+		vty_out(vty, "'show bgp community-list <(1-500)|WORD> detail'\n");
 		zlog_warn("Deprecated option: 'ip show community-list <(1-500)|WORD>' being used");
```

Comment:
I suggest changing this typo and others as well related to this :)

`ip show community-list` --> `show ip community-list`